repo/stable/kafka/versions/0/kafka-frameworkversion.yaml

```diff
@@ -317,7 +312,7 @@ spec:
       strategy: serial
       phases:
         - name: deploy-kafka
-          strategy: parallel
+          strategy: serial
```

Comment:
Why this change?

Reply:
We have it as `serial` here https://github.com/mesosphere/dcos-kafka-service/blob/master/frameworks/kafka/universe/config.json#L83 and here https://github.com/kudobuilder/kudo/blob/master/config/samples/kafka-frameworkversion.yaml#L315

I think this is the right step in light of kep-0006 and think this should be further discussed when looking at more details but for now I think this is the right approach.